http-clients/netty-nio-client/src/main/java/software/amazon/awssdk/http/nio/netty/internal/ChannelAttributeKey.java

```diff
@@ -61,6 +61,11 @@
     static final AttributeKey<Long> EXECUTION_ID_KEY = AttributeKey.newInstance(
             "aws.http.nio.netty.async.executionId");
 
+    /**
+     * Whether the channel is still in use
+     */
+    static final AttributeKey<Boolean> IN_USE = AttributeKey.newInstance("inUse");
```

Comment:
We should namespace this (#669)

Reply:
+1, will fix. How about `AWS_JAVA_SDK_XXX`, I will fix others as well while we can

Reply:
the one above is prefixed with `aws.http.nio.netty.async` we can just follow the same or similar pattern

Reply:
Seems like others are using aws.http.nio.netty.async.

Reply:
Can we still fix other attribute keys now? or it would be backwards incompatible change at this point.

Reply:
These are internal so it should be fine.